config.yaml

```diff
@@ -24,8 +24,26 @@ options:
     type: string
     default: "auto"
     description: |
-      docker runtime to install valid values are "upstream" (docker PPA), "nvidia" (nvidia PPA),
-      "apt" (ubuntu archive), or "auto" (nvidia PPA or ubuntu archive, based on your hardware)
+      Docker runtime to install valid values are "upstream" (docker PPA), "nvidia" (nvidia PPA),
+      "apt" (ubuntu archive), "auto" (nvidia PPA or ubuntu archive, based on your hardware),
+      or "custom" (set repository URL, validation key URL and package name).
```

Comment:
Here I think it would be better to say "(set docker_runtime_repo, docker_runtime_key_url, and docker_runtime_package config options)." Just so it's explicit exactly what needs to be set when using the "custom" option.